### DISCUSSION TITLE
Apple Silicon

### DISCUSSION THREAD

**Discussion by shaunpitchers** [mrklein/openfoam-os-x#84] [Q&A]
Hi mrklein,

I was wondering if the scripts you write are possible to compile OpenFOAM successfully on an M1 pro chip? I have used your scripts for years, which have been a tremendous help. But I recently tried it on my new MacBook and it compiled okay, but I got the following error after trying to run a test command:

Shauns-MacBook-Pro:OpenFOAM-5.x shaunpitchers$ simpleFoam 
dyld[45492]: symbol not found in flat namespace (_MPI_Abort)
Abort trap: 6

The same error was for versions 9 and 10. 

What are your thoughts on this? 

Regards,
Shaun

**Comment by mrklein** [comment]
Hi @spitchers9,

Since I do not have access to M1/M2 Macs, I can only guess what is going on. There was reported issue (https://github.com/mrklein/openfoam-os-x/issues/76) with similar error. And the problem was with MPI library paths not being in `DYLD_LIBRARY_PATH`.
